documentation/src/docs/asciidoc/writing-tests.adoc

```diff
@@ -94,6 +94,16 @@ are `static` methods in the `{Assertions}` class.
 include::{testDir}/example/AssertionsDemo.java[tags=user_guide]
 ----
 
+Junit Jupiter also comes with a few assertion methods that lend themselves well to being
+used in https://kotlinlang.org/[Kotlin]. All JUnit Jupiter Kotlin assertions are top-level
+functions in the `org.junit.jupiter.api` package.
+
+// TODO: Change to using kotlin language highlighting after switch to rouge syntax highlighter
```

Comment:
What is the status of switching to the rouge syntax highlighter?

Reply:
Still blocked by https://github.com/asciidoctor/asciidoctor/issues/1040.